content/source/docs/cloud/agents/index.html.md

```diff
@@ -23,6 +23,15 @@ The agent architecture is pull-based, so no inbound connectivity is required. An
 
 Agents support Terraform versions 0.12 and above. Workspaces configured to use Terraform versions below 0.12 will not be able to select the agent-based execution mode.
 
+### Hardware Requirements
+
+The instance running the agent will have varying resource requirements depending on the workspace being run. Memory, CPU and disk space should be monitored on the agent instance and adjusted accordingly. The specs below are provided as a reference. Depending on the type of environment you deploy to, the name of your instance may vary.
+
+* At least 4GB of free disk space
+  * Each run requires the agent to temporarily store local copies of the tarred repository, extracted repository, state file, any providers or modules, and the Terraform binary itself.
+* At least 2GB of system memory
+* At least 2 CPU cores
```

Comment:
Unless we have proof around the number of CPU cores, I would separate it from the min requirements as a note that calls out Terraform will spread out across cores if they are made available.

Reply:
My logic on that was with 1 core that the agent, terraform main proc, forked procs, and the providers all compete and thrash for CPU and things slowed down a bit when I was playing locally (non cloud stuff).
I think we can remove it - if run on 1 core it works, just slower which should be obvious to most.